src/System.Diagnostics.Process/src/Resources/Strings.resx

```diff
@@ -187,6 +187,9 @@
   <data name="ProcessorAffinityNotSupported" xml:space="preserve">
     <value>Processor affinity for processes or threads is not supported.</value>
   </data>
+  <data name="ProcessStartIdentityNotSupported" xml:space="preserve">
+    <value>Process start with identity is not supported on this platform.</value>
```

Comment:
I like the present comment, Dan's seems easier to understand, with less english words, at least in my head :)